clients/csharp-dotnet-core/voice-assistant-test/tool/AppSettings.cs

```diff
@@ -119,6 +119,11 @@ internal class AppSettings
         /// </summary>
         public string KeywordRecognitionModel { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to enable keyword verification in the SDK.

```

Comment:
From this sentence it sounds like KWV is done in the SDK. I would reword it such that it's clear it's done in the cloud.